src/consensus/aft/raft.h

```diff
@@ -408,6 +419,13 @@ namespace aft
 
     void add_configuration(Index idx, const Configuration::Nodes& conf)
     {
+      std::unique_lock<SpinLock> guard(state->lock, std::defer_lock);
+      if (
+        consensus_type == ConsensusType::BFT && is_follower() &&
```

Comment:
How safe is it to call is_follower() without the state lock at this point?

Reply:
Yes you can change the value of is_follower during a view change when there are no concurrent tx